xcp_d/interfaces/prepostcleaning.py

```diff
@@ -67,48 +79,52 @@ def _run_interface(self, runtime):
         # Check if we need to do anything
         if self.inputs.initial_volumes_to_drop == 0:
             # write the output out
-            self._results['bold_file_dropped_TR'] = self.inputs.bold_file
-            self._results['fmriprep_confounds_file_dropped'
-                          '_TR'] = self.inputs.fmriprep_confounds_file
+            self._results["bold_file_dropped_TR"] = self.inputs.bold_file
+            self._results[
+                "fmriprep_confounds_file_dropped" "_TR"
```

Comment:
This is an odd way to do it. Did you also run a ton of auto-linting for this PR?